tailwind.config.js

```diff
@@ -144,6 +144,18 @@ module.exports = {
               backgroundColor: theme("colors.gray.100"),
               lineHeight: 1.5,
             },
+            code: {
+              backgroundColor: theme("colors.gray.100"),
+              padding: "0.25rem",
+              borderRadius: "3px",
+              margin: "-0.25rem 1px",
+            },
+            "code::before": {
+              content: "",
+            },
+            "code::after": {
+              content: "",
+            },
```

Comment:
I don't believe this is used anywhere. It would be good to delete it then

Reply:
Done - unused code removed